File: javascript/bubblesort.js
function bubbleSort(arr) {
    let n = arr.length;
    let temp;

    for (let i = 0; i < n - 1; i++) {
        for (let j = 0; j < n - i - 1; j++) {
            if (arr[j] > arr[j + 1]) {
                // Swap arr[j] and arr[j + 1]
                temp = arr[j];
                arr[j] = arr[j + 1];
                arr[j + 1] = temp;
            }
        }
    }
}

const arr = [64, 34, 25, 12, 22, 11, 90]; // Example array

console.log("Unsorted array: " + arr.join(", "));

bubbleSort(arr);

console.log("Sorted array in ascending order: " + arr.join(", "));

File: Python/bubblesort.py
def bubble_sort(array):
    n=len(array)

    for i in range(n):
        for j in range(0,n-i-1):
            if array[j]>array[j+1]:
                temp=array[j]
                array[j]=array[j+1]
                array[j+1]=temp

def printArray(array):
    for i in range(len(array)):
        print (array[i],end=' ')
    print()

# Example usage:
array=[10,9,2,120,30,80,21,6,1]
print("Given array is",end="\n")
printArray(array)

# Sort array calling bulle_sort function
bubble_sort(array)
print("Sorted Array is")
printArray(array)